VideoCallKitQuickStart/ViewController.swift

```diff
@@ -86,6 +86,49 @@ class ViewController: UIViewController {
         let tap = UITapGestureRecognizer(target: self, action: #selector(ViewController.dismissKeyboard))
         self.view.addGestureRecognizer(tap)
     }
+    
+    func setupRemoteVideoView() {
```

Comment:
I would add a comment mentioning that this is an example of programmatically creating a `TVIVideoView`.